documentation/identifiers.md

```diff
@@ -78,65 +88,94 @@ There are many different kinds of organisations that give, receive or benefit fr
 * Registered charities
 * Community organisations
```

Comment:
Here do we mean unregistered groups?

Reply:
A more standard term is voluntary and community organisations so I'd use that.